### PR TITLE
Creating automation to help create releases.

### PR DESCRIPTION
**Pull request by n3wscott**
Steps:

1. Run the ./hack/tag-release.sh locally, and push up the tags. 
2. github actions will generate releases from these tags.

Later we can automate the creation of the sub-modules from the parent release.

### CODE REVIEW

File: .github/workflows/draft-release.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    - name: Create Draft Release
+      id: create_draft_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
+        draft: true
```
Comment (slinkydeveloper) [inline .github/workflows/draft-release.yaml:25]: draft release? is it valid?

Comment (n3wscott) [reply .github/workflows/draft-release.yaml:25]: I wanted to have a run with this as a draft before trusting it, it is valid.

Comment (slinkydeveloper) [reply .github/workflows/draft-release.yaml:25]: Ok cool, maybe we should understand how to make it configurable somehow

Comment (slinkydeveloper) [reply .github/workflows/draft-release.yaml:25]: like some fancy regex depending on the tag

Comment (slinkydeveloper) [reply .github/workflows/draft-release.yaml:25]: but you can do it as followup after we enable it for real releases :)

**Pull request merged**